helpers/doesNotNeedPlugin.js

```diff
@@ -5,15 +5,15 @@ const isStaticExportProject = require('./isStaticExportProject')
 const doesSiteUseNextOnNetlify = require('./doesSiteUseNextOnNetlify')
 const hasCorrectNextConfig = require('./hasCorrectNextConfig')
 
-const doesNotNeedPlugin = async ({ netlifyConfig, packageJson }) => {
+const doesNotNeedPlugin = async ({ netlifyConfig, packageJson, utils }) => {
```

Comment:
[dust] should we pass only `failBuild`

Reply:
i was trying to keep it on the same level as netlifyConfig and packageJson, as in they're all things that get passed to the different event handlers (i.e onBuild) so i think it's ok? will keep in mind tho going forward